doc/getting-started/understanding-config-files.md

```diff
@@ -21,6 +21,64 @@ Tells your node to which nodes in the network it should talk to. A minimal versi
 
 Your __block-producing__ node must __ONLY__ talk to your __relay nodes__, and the relay node should talk to other relay nodes in the network. Go to our telegram channel to find out IP addresses and ports of peers.
 
+#### The P2P topology.json file
+
+Tells your node to which nodes in the network it should talk to. A minimal version of this file looks like this:
+
+```json
+{
+  "LocalRoots": {
+    "groups": [
+      {
+        "localRoots": {
+          "addrs": [
+            {
+              "addr": "x.x.x.x",
+              "port": 3001
+            }
+          ],
+          "advertise": false
+        },
+        "valency": 1
+      }
+    ]
+  },
+  "PublicRoots": [
+    {
+      "publicRoots" : {
+        "addrs": [
+          {
+            "addr": "y.y.y.y",
+            "port": 3002
+          }
+        ],
+        "advertise": false
+      }
+    }
+  ],
+  "useLedgerAfterSlot": 0
+}
+```
+
+* The main difference between `LocalRoots` and `PublicRoots` is that with the former
+    you can specify different groups with different valencies. That can be useful to
+    inform your node of different targets withing a group to achieve. `LocalRoots`
+    is for peers which the node always should have as hot, such as their own block producer.
+    `PublicRoots` represent a source of fallback peers, a source of peers to be used if peers
+    from the ledger (`useLedgerAfterSlot`) is disabled or unavailable.
+
+* This means that your node will contact node at ip `x.x.x.x` on `port 3001`, and resolve
+    dns domain `y.y.y.y` (assuming they are), and try to maintain a connection with at least `1` of the
+    resolved ips.
+
+* `valency` tells the node how many connections your node should try to pick from the given group. If a dns address is given, valency governs to how many resolved ip addresses should we maintain active (hot) connection.
+
+Your __block-producing__ node must __ONLY__ talk to your __relay nodes__, and the relay node should talk to other relay nodes in the network.
+
+You __can__ tell the node that the topology configuration file changed by sending a SIGHUP
+signal to the `cardano-node` process, e.g. `pkill -HUP cardano-node`. After receiving the
+signal, `cardano-node` will re-read the file and restart all dns resolution. Please
+**note** that this only applies to the topology configuration file!
```

Comment:
Does this work or is there an equivalent for Windows?

Reply:
Is `*nix` only, and unlikely needed on Windows.